alerting/migration/1738249210356-create-anticipatory-action-alerts-table.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CreateAnticipatoryActionAlertsTable1738249210356
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`CREATE TABLE "anticipatory_action_alerts" (
```

Comment:
@wadhwamatic we decided to store the alerts in a new table since the behavior was quite different.
This will let us control mailing lists and target urls more easily. We made it so that it can be used for other countries in the future